src/nvim/ex_docmd.h

```diff
@@ -19,6 +19,20 @@
 #define EXMODE_NORMAL           1
 #define EXMODE_VIM              2
 
+/// The scope of a command.
+///
+/// The lower a number, the deeper the scope.
+typedef enum {
+  kCdScopeWindow,  ///< Affects one window.
+  kCdScopeTab,     ///< Affects on tab page.
+  kCdScopeEditor,  ///< Affects the entire instance of NeoVim.
```

Comment:
Which docs? I ran `ack kCdScopeGlobal` and it didn't give me any results. I don't really care about the name, but "global" sounds kind of nebulous, while Editor makes it clear that it refers the the Neovim editor. How about `kCdScopeInstance` or `kCdScopeApplication`?